bHY6yh-d56743/cod-vyg67d__bHY6yh-d56743.xml

```diff
@@ -0,0 +1,453 @@
+            </p>
+            <p xml:id="bHY6yh-d56743-d1e499"> 
+               ¶ <ref xml:id="bHY6yh-d56743-Rd1e581">Idem de tranquillitate animi</ref> ¶ <quote xml:id="bHY6yh-d56743-Qd1e501">pu<lb ed="#O" break="no"/>to 
+               <pb ed="#O" n="124-r"/>
+               <cb ed="#O" n="a"/>
+               <lb ed="#O"/>multos potuisse ad sapientiam perue<lb ed="#O" break="no"/>nire 
+               nisi putassent se peruenisse. ¶ 
+               <lb ed="#O"/>Onerat discente turba librorum non 
+               <lb ed="#O"/>instituit multoque satius est paucis 
+               <lb ed="#O"/>te auctoribus tradere quam errare per mul<lb ed="#O" break="no"/>tos.</quote>
+            </p>
+            <p xml:id="bHY6yh-d56743-d1e516">
+               <!--line bumps down-->
+               <lb ed="#O"/><ref xml:id="bHY6yh-d56743-Rd1e601">IU<!--line bumps up-->uenalis. libro 5. et fi. saty. 2.</ref> ¶ <quote xml:id="bHY6yh-d56743-Qd1e520">Do<!--line bumps back down-->ciles imitandis ¶ Turpibus 
+               <lb ed="#O"/>ac prauis omnes sumus.</quote>
+            </p>
+            <p xml:id="bHY6yh-d56743-d1e526">
+               <!--paragraph moved down to lines 294-297--> 
+            </p>
+            <p xml:id="bHY6yh-d56743-d1e538"> 
+               <!--paragraph moved down to lines 301-303-->
+            </p>
+            <p xml:id="bHY6yh-d56743-d1e637">
+               <!--line bumps down-->
+               <lb ed="#O"/><ref xml:id="bHY6yh-d56743-Rd1e641">UAlerius maximus libro 4. ti. <!--line bumps up--> de stu<!--line bumps back down-->dio et 
+               <lb ed="#O"/>industria.</ref> ¶ <quote xml:id="bHY6yh-d56743-Qd1e646">Democritus cum di<lb ed="#O" break="no"/>uitiis 
+               censeri posset que tante fuerunt 
+               <lb ed="#O"/>Ut pater eius serxis exercitui exemplum 
+               <lb ed="#O"/>dare ex facili potuerit quo magis 
+               <lb ed="#O"/>uacuo animo studiis litterarum esset operatus 
+               <lb ed="#O"/>parua admodum summa retenta 
+               <lb ed="#O"/>Patrimonium suum patrie donauit.</quote>
+            </p>
+            <p>
+               <!--following paragraph moved down from line 276-->
+               <lb ed="#O"/><ref xml:id="bHY6yh-d56743-Rd1e613">LEx dige. ti. de fideicommissa. liberta<lb ed="#O" break="no"/>tibus. 
+               l. apud iulianum.</ref> ¶ <quote xml:id="bHY6yh-d56743-Qd1e530">Cupi<lb ed="#O" break="no"/>ditas 
+               discendi est optima ratio 
+               <lb ed="#O"/>uiuendi.</quote>
+            </p>
+            <p> 
+               <!--following paragraph moved down from line 279-->
+               ¶ <ref xml:id="bHY6yh-d56743-Rd1e626">lex cod. libro 10. t. de ex<lb ed="#O" break="no"/>cusatur. 
+               artificium. libro 1.</ref> ¶ <quote xml:id="bHY6yh-d56743-Qd1e540">In addipiscen<lb ed="#O" break="no"/>dis 
+               artibus ocium est accommodandum.</quote>
+            </p>
```

Comment:
is the issue here that the paragraphs are just out of order. If they are out of order, we can move them around, as long as we retain the paragraph id. So I would suggest the following: 

Please double check my suggestion before accepting. Make sure I'm correctly understanding what's going on. If I'm not let's meet on zoom to look at it together.

```suggestion

               <!--paragraph bHY6yh-d56743-d1e526 appears below bHY6yh-d56743-d1e637--> 
               <!--paragraph bHY6yh-d56743-d1e538 appears below bHY6yh-d56743-d1e637-->
            <p xml:id="bHY6yh-d56743-d1e637">
               <!--line bumps down-->
               <lb ed="#O"/><ref xml:id="bHY6yh-d56743-Rd1e641">UAlerius maximus libro 4. ti. <!--line bumps up--> de stu<!--line bumps back down-->dio et 
               <lb ed="#O"/>industria.</ref> ¶ <quote xml:id="bHY6yh-d56743-Qd1e646">Democritus cum di<lb ed="#O" break="no"/>uitiis 
               censeri posset que tante fuerunt 
               <lb ed="#O"/>Ut pater eius serxis exercitui exemplum 
               <lb ed="#O"/>dare ex facili potuerit quo magis 
               <lb ed="#O"/>uacuo animo studiis litterarum esset operatus 
               <lb ed="#O"/>parua admodum summa retenta 
               <lb ed="#O"/>Patrimonium suum patrie donauit.</quote>
            </p>
            <p xml:id="bHY6yh-d56743-d1e526">
               <lb ed="#O"/><ref xml:id="bHY6yh-d56743-Rd1e613">LEx dige. ti. de fideicommissa. liberta<lb ed="#O" break="no"/>tibus. 
               l. apud iulianum.</ref> ¶ <quote xml:id="bHY6yh-d56743-Qd1e530">Cupi<lb ed="#O" break="no"/>ditas 
               discendi est optima ratio 
               <lb ed="#O"/>uiuendi.</quote>
            </p>
            <p xml:id="bHY6yh-d56743-d1e538">  
               ¶ <ref xml:id="bHY6yh-d56743-Rd1e626">lex cod. libro 10. t. de ex<lb ed="#O" break="no"/>cusatur. 
               artificium. libro 1.</ref> ¶ <quote xml:id="bHY6yh-d56743-Qd1e540">In addipiscen<lb ed="#O" break="no"/>dis 
               artibus ocium est accommodandum.</quote>
            </p>
```